### ISSUE TITLE
Add CI for the examples

### ISSUE THREAD

**Issue by mdiazmel** [jean-zay-users/jean-zay-doc#48]
It should be nice to perform some kind of automated testing on the examples.

**Comment by lesteve** [comment]
Yeah I agree this would be ideal but I don't know how practical this would be, unless the discussion about CI on Jean Zay actually turns into something real (probably not in the near future though).

For now, I think I am fine with having examples that we fix when people report problems.

**Comment by mdiazmel** [comment]
The idea is to have a nice green marker when somebody submit a new PR, mainly when adding new examples, in order to test that nothing is broken and that the new code works well (i.e., that the slurm job van be submitted to Jean Zay).
I put it here to force myself to do some experiments and, eventually, to discuss about it with the JZ's admins ;)
